setup/setuputils.class.inc.php

```diff
@@ -897,9 +897,16 @@ public static function CheckDiskSpace($sDir)
 		return $f;
 	}
 
+    /**
+     * @param float $fBytes size in raw bytes, for example 162594750464.0
+     * @return string formatted string, for example "161.62 GB"
+     *
+     * @link https://en.wiktionary.org/wiki/petabyte petabyte PB
+     * @link https://en.wiktionary.org/wiki/exabyte#English exabyte EB
+     */
 	public static function HumanReadableSize($fBytes)
 	{
-		$aSizes = array('bytes', 'Kb', 'Mb', 'Gb', 'Tb', 'Pb', 'Hb');
+		$aSizes = array('Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB');
```

Comment:
Actually it can be lowercase. Also, the k in kilobytes is also lowercase
```suggestion
		$aSizes = array('bytes', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB');
```
See https://en.wikipedia.org/wiki/Kilobyte

Reply:
Yes indeed, thanks !
About bytes VS Bytes : 
https://en.wiktionary.org/wiki/byte#English
https://en.wiktionary.org/wiki/Byte

Reply:
Fixed in 590c8ce3